include/onnxruntime/core/graph/graph.h

```diff
@@ -872,11 +872,14 @@ class Graph {
 
   /** Gets the mutable NodeArg with the provided name.
   @returns Pointer to NodeArg if found, nullptr if not. */
-  NodeArg* GetNodeArg(const std::string& name) {
+  NodeArg* GetNodeArg(const std::string& name, bool check_ancestors = false) {
```

Comment:
nit: Add docs for this parameter?

Reply:
It would be better to call GetNodeArgIncludingParentGraphs instead of changing this function signature.